build/quarks-secret/quarks-values.yaml

```diff
@@ -0,0 +1,9 @@
+fullnameOverride: cf-quarks-secret
+
+image:
+  repository: quarks-secret
+  org: us.gcr.io/cf-security-credhub-main
+  tag: latest
+
+global:
+  monitoredID: cf
```

Comment:
Do we need a different default here than the release fullname? We think it's fine to let the monitoredId be an arbitrary value at the helm template step so that we can consolidate the monitoredID configuration and namespace labelling all in a focused ytt overlay file.

Reply:
ya that should be fine, we'll make that change